src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/PaperCaseCreateContestedAboutToSubmitHandlerTest.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.controllers.GenericAboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseRole;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.YesOrNo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.CaseFlagsService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.IdamService;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.NO_VALUE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.YES_VALUE;
+
+@ExtendWith(MockitoExtension.class)
+class PaperCaseCreateContestedAboutToSubmitHandlerTest extends BaseHandlerTestSetup {
+
+    private static final String CONTESTED_HWF_JSON = "/fixtures/contested/hwf.json";
+    private static final String CONTESTED_VALIDATE_HEARING_SUCCESSFULLY_JSON = "/fixtures/contested/validate-hearing-successfully.json";
+    private static final String AUTH_TOKEN = "tokien:)";
+    private PaperCaseCreateContestedAboutToSubmitHandler handler;
+    @Mock
+    private IdamService idamService;
+    @Mock
+    private CaseFlagsService caseFlagsService;
+
+    @BeforeEach
+    void setup() {
+        FinremCaseDetailsMapper finremCaseDetailsMapper = new FinremCaseDetailsMapper(new ObjectMapper().registerModule(new JavaTimeModule()));
+        handler = new PaperCaseCreateContestedAboutToSubmitHandler(finremCaseDetailsMapper, caseFlagsService, idamService);
+    }
+
+    @Test
```

Comment:
When you're writing tests for the canHandle method you should use

https://github.com/hmcts/finrem-case-orchestration-service/blob/master/src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/test/Assertions.java

See here for example usage:
https://github.com/hmcts/finrem-case-orchestration-service/blob/1428fdb8d0471c84af48a926f0ba6b3814f2ac00/src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/draftorders/upload/UploadDraftOrderAboutToStartHandlerTest.java#L59